packages/now-next/src/index.ts

```diff
@@ -433,13 +433,30 @@ export const build = async ({
     {}
   );
 
-  const staticDirectoryFiles = onlyStaticDirectory(
-    includeOnlyEntryDirectory(files, entryDirectory),
-    entryDirectory
+  const entryDirectoryFiles = includeOnlyEntryDirectory(files, entryDirectory);
+  const staticDirectoryFiles = filesFromDirectory(
+    entryDirectoryFiles,
+    path.join(entryDirectory, 'static')
+  );
+  const publicDirectoryFiles = filesFromDirectory(
+    entryDirectoryFiles,
+    path.join(entryDirectory, 'public')
+  );
+  const publicFiles = Object.keys(publicDirectoryFiles).reduce(
+    (mappedFiles, file) => ({
+      ...mappedFiles,
+      [file.replace('public/', '')]: publicDirectoryFiles[file],
```

Comment:
```suggestion
      [file.replace('public' + path.sep, '')]: publicDirectoryFiles[file],
```

Should be this for cross-platform, right?

Reply:
Maybe this would just be safer:

```suggestion
      [file.replace(/public[/\\]+/, '')]: publicDirectoryFiles[file],
```